packages/css/button.css

```diff
@@ -1,202 +1,158 @@
+  --dsc-button-primary-background--active: var(--ds-color-accent-base-active);
+  --dsc-button-primary-background--hover: var(--ds-color-accent-base-hover);
+  --dsc-button-primary-background: var(--ds-color-accent-base-default);
+  --dsc-button-primary-color: var(--ds-color-accent-contrast-default);
+  --dsc-button-secondary-background--active: var(--ds-color-accent-surface-hover);
+  --dsc-button-secondary-background--hover: var(--ds-color-accent-surface-default);
+  --dsc-button-secondary-background: transparent;
+  --dsc-button-secondary-border-color: var(--ds-color-accent-border-strong);
+  --dsc-button-secondary-color--active: var(--ds-color-accent-text-default);
+  --dsc-button-secondary-color--hover: var(--ds-color-accent-text-default);
+  --dsc-button-secondary-color: var(--ds-color-accent-text-subtle);
+  --dsc-button-tertiary-background--active: var(--ds-color-accent-surface-hover);
+  --dsc-button-tertiary-background--hover: var(--ds-color-accent-surface-default);
+  --dsc-button-tertiary-background: transparent;
+  --dsc-button-tertiary-color--active: var(--ds-color-accent-text-default);
+  --dsc-button-tertiary-color--hover: var(--ds-color-accent-text-default);
+  --dsc-button-tertiary-color: var(--ds-color-accent-text-subtle);
 
-  display: flex;
   align-items: center;
-  border: var(--ds-border-width-default) solid transparent;
-  background-color: var(--dsc-btn-primary-background);
-  color: var(--dsc-btn-primary-color);
-  min-width: 2.5em;
-  padding: var(--dsc-btn-padding);
+  background: var(--dsc-button-background);
+  border-radius: var(--ds-border-radius-md);
+  border: var(--ds-border-width-default) solid var(--dsc-button-border-color, transparent);
   box-sizing: border-box;
+  color: var(--dsc-button-color);
   cursor: pointer;
-  font-family: inherit;
+  display: flex;
+  font: inherit;
+  gap: var(--dsc-button-gap);
   justify-content: center;
+  min-height: var(--dsc-button-size);
+  min-width: var(--dsc-button-size); /* 2.5em; /* TODO: Why not ds-spacing here? */
+  outline: none;
+  padding: var(--dsc-button-padding);
   text-align: inherit;
   text-decoration: none;
-  position: relative;
-  border-radius: var(--ds-border-radius-md);
-  min-height: var(--ds-sizing-10);
-}
 
-.ds-btn svg {
-  overflow: visible;
-}
-
-.ds-btn:disabled,
-.ds-btn[aria-disabled='true'] {
-  cursor: not-allowed;
-  opacity: var(--ds-disabled-opacity);
-}
+  & svg { overflow: visible } /* TODO: Add comment about why? */
 
-.ds-btn--sm {
-  --dsc-btn-padding: var(--ds-spacing-2) var(--ds-spacing-3);
-
-  gap: var(--ds-sizing-1);
-  min-height: var(--ds-sizing-10);
-}
+  /**
+    * Sizes
+    */
+  &[data-size="sm"] {
+    --dsc-button-padding: var(--ds-spacing-2) var(--ds-spacing-3);
+    --dsc-button-gap: var(--ds-sizing-1);
+    --dsc-button-size: var(--ds-sizing-10); /* TODO: 40px, enough click surface? Yes accordting to Nielsen Norman group https://www.nngroup.com/articles/touch-target-size/ */
```

Comment:
https://github.com/digdir/designsystemet/issues/537

Reply:
yeah, but not sure this is 100% correct - we should discuss with design ☺️

Reply:
Settles with Lasse; we do not need to expand the click surface here :)